streaming/stream.conf

```diff
@@ -26,6 +26,8 @@
     #             IPv6 IPs should be given with brackets [ip:address]
     # INTERFACE = the network interface to use (only for IPv6)
     # PORT      = the port number or service name (/etc/services)
+    # SSL       = when this word appear at the end of the destination string
+    #             the Netdata will do encrypt connection with the master.
```

Comment:
When I put the :SSL on my Fedora slave, I got this: 2019-05-22 17:31:05: netdata ERROR : STREAM_SENDER[fedora] : Failed to allocate SSL structure.
I haven't given it any ssl certificate paths and I shouldn't have to, correct?
I mean, only the server needs them, not the client

Reply:
You are correct, it is not necessary to have any certificate and key in the slave side, it was developer mistake, I copied and paste a code, but I had to change and I did not do! It was fixed.